azure-pipelines.yml

```diff
@@ -12,7 +8,7 @@ trigger:
 
 # Build only for PRs for master branch
 pr:
-  autoCancel: true
+  autoCancel: "true"
```

Comment:
Intuitively (and from https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml ), this change should not be necessary? This also applies for further booleans in this document.

Reply:
Shouldn't be necessary but my vscode azure pipeline validator plugin was having a cow about it, so this was more about appeasing my editor 😆

Reply:
Actually just changed that, editor isn't complaining, either it got an update or I remembered wrong... will update the PR will that change.